tests/cli/cases/empty-parsable-changelog.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 Resin.io
```

Comment:
Should this copyright notice be Balena? Since Resin doesn't exists since 2018

Reply:
Updated :P